src/locales/en.json

```diff
@@ -49,6 +49,15 @@
     "rename": "Rename",
     "close": "Close"
   },
+  "deleteconfirmation": {
```

Comment:
I don't know if it should be `"deletion_confirmation"` or `"deletionconfirmation"`.
I let others decide.